released/packages/coq-io-system-ocaml/coq-io-system-ocaml.2.3.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "dev@clarus.me"
+homepage: "https://github.com/clarus/io-system-ocaml"
+dev-repo: "git+https://github.com/clarus/io-system-ocaml.git"
+bug-reports: "https://github.com/clarus/io-system-ocaml/issues"
+authors: ["Guillaume Claret"]
+license: "MIT"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "lwt" {>= "2.4.7"}
+  "ocaml" {>= "4.02.0"}
+  "ocamlbuild"
```

Comment:
Does this really depend on `ocamlbuild` during runtime? If not, please use `"ocamlbuild" {build}`. Same question about `ocamlfind`.

Reply:
Indeed, did the fix.